apps/studio/src/lib/db/clients/cockroach.ts

```diff
@@ -26,6 +26,61 @@ export class CockroachClient extends PostgresClient {
     return [];
   }
 
+  async listTableColumns(table?: string, schema: string = this._defaultSchema): Promise<ExtendedTableColumn[]> {
```

Comment:
I think you'll have to do the same thing for `redshift.ts` also.

Reply:
Done. I think we need to add integration test at a separate PR(not in this task).

Reply:
Yeah, we don't really have a way to run Redshift integration tests right now, but that'll come in the future